docs/build/hosted-lotus-node.md

```diff
@@ -0,0 +1,643 @@
+---
+title: Hosted Lotus Node
+description: As the fundamental component of Filecoin Network, lotus node offers abundant features to developers via command-line as well as RPC API methods. 
+breadcrumb: 'Lotus Node'
+---
+
+# {{ $frontmatter.title }}
+
+{{ $frontmatter.description }} 
+
+In the Filecoin ecosystem, there are various node solutions to connect to the Filecoin network and provide  functions via RPC API calls. Developers can either install and run their own lotus node locally to interact with the Filecoin network, or connect to other lotus nodes hosted by Glif or Infura.
+
+Developers can also use the API client libraries to utilize the features of Filecoin Network since those libraries take care of the low-level logic of connecting to lotus node, requesting and handing the response via simple calls, etc. Currently, there are many libraries available:
+
++ [filecoin.js](https://filecoin-shipyard.github.io/filecoin.js/)
++ [filecoin-js-signer](https://github.com/blitslabs/filecoin-js-signer)
++ [Filecoin Signing Tools](https://github.com/Zondax/filecoin-signing-tools)
+
+In this tutorial, we will run a  **local lotus node** and use **filecoin.js** library to build examples to walk you through: 
```

Comment:
Bolding should be reserved for when the user/reader has to look for something, or click on something. https://docs.filecoin.io/community/contribute/grammar-formatting-and-style/#bold-text